
global_costmap:
  global_frame: map
  robot_base_frame: base_footprint

  update_frequency: 0.1
  publish_frequency: 0.1
  transform_tolerance: 0.5
  resolution: 0.05
 
  # footprint: [[-0.105, -0.105], [-0.105, 0.105], [0.041, 0.105], [0.041, -0.105]]
  robot_radius: 0.25

  inflation_layer:
    inflation_radius: 1.62
    cost_scaling_factor: 2.58

  plugins:
    - {name: static_layer,    type: "costmap_2d::StaticLayer"}
    - {name: inflation_layer, type: "costmap_2d::InflationLayer"}

